Source/Glide64/Config.cpp

```diff
@@ -1310,3 +1310,28 @@ void CALL DllAbout ( HWND hParent )
   hostWindow = NULL;
 #endif
 }
+
+void general_setting(short setting_ID, const char * name, unsigned int value)
+{
+    RegisterSetting(
+        setting_ID,
+        Data_DWORD_General,
+        name,
+        NULL,
+        value,
+        NULL
+    );
+    return;

```

Comment:
haven't been doing it in every pull request though...maybe in this particular commit it triggered me to do it because of this:
```c
void general_setting(short setting_ID, const char * name, unsigned int value)
{
    RegisterSetting(
        ... x ...
```
... instead of ...
```c
void general_setting(short setting_ID, const char * name, unsigned int value)
{
    RegisterSetting(... x ...);
```
which caused me to remember my habit in this PR.  IDK, doesn't really matter, I'd have taken it back out if you wanted me to I guess.